src/backend/types.ml

```diff
@@ -296,8 +324,333 @@ module Competition = struct
           ~examples:[`String "P4T"];
         "kind", ref Kind.ref;
         "category", ref Category.ref;
-        "leaders_count", obj @@ S.make_schema () ~typ:int;
-        "followers_count", obj @@ S.make_schema () ~typ:int;
+        "n_leaders", obj @@ S.make_schema () ~typ:int;
+        "n_follows", obj @@ S.make_schema () ~typ:int;
+      ]
+end
+
+(* Artefact *)
+(* ************************************************************************* *)
+
+module YanCriterionWeight = struct
+  type t = Ftw.Ranking.Algorithm.yan_weight [@@deriving yojson]
+
+  let ref, schema =
+    make_schema ()
+      ~name:"YanCriterionWeight"
+      ~typ:(Obj Object)
+      ~properties:[
+        "yes", obj @@ S.make_schema ()
+          ~typ:int;
+        "alt", obj @@ S.make_schema ()
+          ~typ:int;
+        "no", obj @@ S.make_schema ()
+          ~typ:int;
```

Comment:
maybe we could specify here that these are meant to be positive ? It's not yet checked, but it seems like a pertinent amount of doc for the openapi docs ?